acceptance/acceptance_test.go

```diff
@@ -229,1905 +99,1417 @@ func testWithoutSpecificBuilderRequirement(
 	t *testing.T,
 	when spec.G,
 	it spec.S,
-	packFixturesDir, packPath string,
-	bpVersion *api.Version,
+	assetManager managers.AssetManager,
+	registryConfig *h.TestRegistryConfig,
 ) {
 	var (
-		bpDir    = buildpacksDir(*bpVersion)
-		packHome string
+		pack   *components.PackExecutor
+		assert = assertions.NewAssertionManager(t, dockerCli)
 	)
 
-	// subjectPack creates a pack `exec.Cmd` based on the current configuration
-	subjectPack := func(name string, args ...string) *exec.Cmd {
-		return packCmd(packHome, packPath, name, args...)
-	}
-
 	it.Before(func() {
-		var err error
-		packHome, err = ioutil.TempDir("", "buildpack.pack.home.")
-		h.AssertNil(t, err)
+		pack = managers.NewPackExecutor(t, assetManager, managers.Current, registryConfig, assert)
 	})
 
 	it.After(func() {
-		h.AssertNil(t, os.RemoveAll(packHome))
+		pack.Cleanup()
 	})
 
 	when("invalid subcommand", func() {
 		it("prints usage", func() {
-			output, err := h.RunE(subjectPack("some-bad-command"))
-			h.AssertNotNil(t, err)
-			h.AssertContains(t, output, `unknown command "some-bad-command" for "pack"`)
-			h.AssertContains(t, output, `Run 'pack --help' for usage.`)
+			output, err := pack.RunWithCombinedOutput("some-bad-command")
+			assert.NotNil(err)
+
+			assert.Contains(output, `unknown command "some-bad-command" for "pack"`)
+			assert.Contains(output, `Run 'pack --help' for usage.`)
 		})
 	})
 
 	when("suggest-builders", func() {
 		it("displays suggested builders", func() {
-			output := h.Run(t, subjectPack("suggest-builders"))
-
-			h.AssertContains(t, output, "Suggested builders:")
-			h.AssertMatch(t, output, `Google:\s+'gcr.io/buildpacks/builder'`)
-			h.AssertMatch(t, output, `Heroku:\s+'heroku/buildpacks:18'`)
-			h.AssertMatch(t, output, `Paketo Buildpacks:\s+'gcr.io/paketo-buildpacks/builder:base'`)
-			h.AssertMatch(t, output, `Paketo Buildpacks:\s+'gcr.io/paketo-buildpacks/builder:full-cf'`)
-			h.AssertMatch(t, output, `Paketo Buildpacks:\s+'gcr.io/paketo-buildpacks/builder:tiny'`)
+			output := pack.SuccessfulRunWithOutput("suggest-builders")
```

Comment:
When I see `SuccessfulRunWithOutput` my first thought is that "suggest-builders" is the output we're expecting to receive. I wonder if we could just call this `SuccessfulRun`.

Reply:
@dfreilich had a similar concern, so this is obviously a problem. I added some commentary to his [comment](https://github.com/buildpacks/pack/pull/673#discussion_r442904946).

I really want to capture the intent that there is a return value, and it is the output. I wouldn't expect `SuccessfulRun` to return anything.